docs/core/versions/selection.md

```diff
@@ -36,6 +36,10 @@ On rare occasions, you may need to use an earlier version of the SDK. You specif
 
 *global.json* can be placed anywhere in the file hierarchy. The CLI searches upward from the project directory for the first *global.json* it finds. You control which projects a given *global.json* applies to by its place in the file system. The .NET CLI searches for a *global.json* file iteratively navigating the path upward from the current working directory. The first *global.json* file found specifies the version used. If that version is installed, that version is used. If the SDK specified in the *global.json* is not found, the .NET CLI rolls forward to the latest SDK installed. Roll-forward is the same as the default behavior, when no *global.json* file is found.
 
+> [!TIP]
+> We recommend that you add a *global.json* file to the root of your drive set to the latest **supported** SDK version.
+> That way, you ensure that all your projects will use the latest **supported** SDK version by default and not the latest **installed** version. Whenever a new supported version is released, then you can just edit that file to update the SDK version.
```

Comment:
@mairaw I don't know if the following remarks are self-evident or worth to be included in the text (so, additional edits might not be required):
- How do I know the latest **supported** SDK version? Is the [download page](https://dotnet.microsoft.com/download) a reliable source?
- I've wondered a little bit about the origin of this tip. Why not to use the latest installed version? Then I've realized it's related to the info at the line 31 of this document: "The latest version of the .NET Core SDK is a preview version." Is it worth to extend the sentence in the tip the following way:
  > ...and not the latest **installed** version, which might be a preview version.

Reply:
I think it is indeed worth extending indeed @pkulikov. This page seems to label the versions nicely: https://dotnet.microsoft.com/download/archives.

@KathleenDollard @coolcsh can you confirm whether the recommendation to use the current version or LTS?